docs/StandaloneConfiguration.md

```diff
@@ -1,8 +1,44 @@
 # RepoSense - Guide to Setup Standalone Configuration
-
+### Quick Start
 1. Add [_reposense/config.json](../_reposense/config.json) to the root of your repository.
-1. Fill it up with author information, such as the GitHub Id, Display Name (Optional) and the [Author Names](UserGuide.md#git-author-name) (Optional).
+1. Fill it up with author information, such as the GitHub Id, Display Name (Optional), [Author Names](UserGuide.md#git-author-name)  (Optional), [Ignore Glob List](UserGuide.md#csv-config-file) (Optional).
+
+### Detailed explanation of the structure of repo-config.json
+
+#### Typical format of repo-config.json
+```
+{
+  "ignoreGlobList": ["**.dat", "**.js"],    
+  "authors": [                              
+  {                                            
+    "githubId": "alice",                       
+    "displayName": "Alice T.",
+    "authorNames": ["AT", "A"],
+    "ignoreGlobList": ["**.css"]
+  },
+  {
+    "githubId": "bob"
+  }
+ ]
```

Comment:
just realized this, the indentation seems to be incorrect.
Please follow the same format as https://github.com/reposense/RepoSense/blob/master/_reposense/config.json